src/neural_network/neuron.rs

```diff
@@ -95,20 +95,20 @@ where
     #[replace_numeric_literals(T::from(literal).unwrap())]
     #[inline]
     pub fn get_value(&mut self) -> T {
-        let update_gate = fast_sigmoid(self.update);
-        let reset_gate = fast_sigmoid(self.reset);
-        let current_memory = fast_tanh(self.input + self.memory * reset_gate);
```

Comment:
We should have options for tanh,sigmoid and relu for all possible activations as it wil bias the output